starter_code/elevator.js

```diff
@@ -2,18 +2,31 @@ class Elevator {
   constructor(){
     this.floor      = 0;
     this.MAXFLOOR   = 10;
-    this.requests   = [];
+    this.requests   = [Person];
   }
 
-  start() { }
-  stop() { }
+  start() {
+    setInterval(update(), 1000);
```

Comment:
Use arrow functions, you may be loosing context in here
```
 this.interval = setInterval( () => { this.update() }, 1000 );
```

this.interval should be defined in the constructor